.github/workflows/publish-gh-pages.yml

```diff
@@ -18,7 +20,32 @@ env:
   TURBO_TEAM: ${{ secrets.TURBO_TEAM }}
 
 jobs:
-  compile-github-pages-and-publish:
+  compile-v1-docs:
```

Comment:
This new GitHub pages script is supposed to:

* checkout the `maintenance/v1.x` branch, build the docs, and upload them as a built artifact
* checkout the default branch, build all the rest of the things under `compile:ghpages`, and upload them as a built artifact
* download the build artifacts and publish